subprojects/packagefiles/speexdsp/meson_options.txt

```diff
@@ -0,0 +1,2 @@
+option('use-fixed-point', type : 'boolean', value : false, description : 'Compile fixed-point library')
+option('with-example-programs', type : 'boolean', value : false, description : 'Build example programs')
```

Comment:
Other wraps use an option called simply `examples` for this. It would be nice to keep consistency, unless there are arguments for deviating from it.

Reply:
Switched to `examples`, thanks.